src/main/java/org/jenkinsci/plugins/workflow/flow/DurabilityHintProvider.java

```diff
@@ -0,0 +1,36 @@
+package org.jenkinsci.plugins.workflow.flow;
+
+import hudson.ExtensionList;
+import hudson.ExtensionPoint;
+import hudson.model.Item;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+
+/**
+ * Provides a way to indirectly register durability settings to apply to pipelines.
+ */
+public interface DurabilityHintProvider extends ExtensionPoint {
+
+
+    public int ordinal();
```

Comment:
Unnecessary—this is defined in `Extension`.

Reply:
Yes, I'm aware -- I want this to be separate from extension loading.

Reply:
Huh? Why?